metadata-ws/src/test/resources/features/study.feature

```diff
@@ -1011,3 +1011,9 @@ Feature: study object
 
     When I request PATCH STUDY with content {""}
     Then the response code should be 400
+
+
+  Scenario: Trying to create a study with an invalid accession name (containing a comma symbol) must fail
```

Comment:
It would be good to include another one to check it fails when the accession has a `dot`

Reply:
Thank you for a suggestion, addressed in https://github.com/EBIvariation/metadata-ws/pull/84/commits/c2233c1301e04bee15af6874c318e29ee1748874